zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -206,6 +206,12 @@ angular.module('zeppelinWebApp')
       $scope.paragraph.result = data.paragraph.result;
       $scope.paragraph.settings = data.paragraph.settings;
 
+      // when paragraph finishes, activate blinking tab notification if user is not focusing the the tab.
+      if (($scope.paragraph.status === 'FINISHED' || $scope.paragraph.status === 'ERROR' || $scope.paragraph.status === 'ABORTED') && 
```

Comment:
this should be ``if (statusChanged && ($rootScope.windowFocus === false))``, otherwise every paragraph will send the event